### DISCUSSION TITLE
Microservices communication with kafka cluster

### DISCUSSION THREAD

**Discussion by shreyasarani23** [strimzi/strimzi-kafka-operator#4942] [General]
I have microservices running as pods and I want my microservices to communicate to the kafka cluster so how can I achieve this?
Is it by specifying  service name of bootstrap server with port ex:-my-kafka-cluster-kafka-bootstrap:9092 or is there any other way?

Suppose if my microservices wants to communicate with particular kafka broker say my-kafka-cluster-kafka-0 then how this can be achieved?

**Comment by scholzj** [comment]
Please read the documentation. It is all described there. Your apps should connect to the bootstrap service.

**Reply by shreyasarani23** [reply]
Thanks @scholzj
